include/deal.II/matrix_free/fe_evaluation.h

```diff
@@ -4859,7 +4945,9 @@ namespace internal
   const std::vector<ArrayView<const typename VectorType::value_type>> *
   get_shared_vector_data(VectorType &vec /*other parameters?*/)
   {
-    return &vec.shared_vector_data();
+    // return &vec.shared_vector_data();
+    (void)vec;
+    return nullptr;
```

Comment:
I am returning a `nullptr` for all vectors for now to prevent that the above code is called. The above code can be only called once #11074 and #11092.